python/ray/tune/trial_runner.py

```diff
@@ -27,7 +27,7 @@ class TrialRunner(object):
 
     While Ray itself provides resource management for tasks and actors, this is
     not sufficient when scheduling trials that may instantiate multiple actors.
-    This is because if insufficient resources are available, concurrent agents
+    This is because if insufficient resources are available, concurrent runners
```

Comment:
why runner? why not just call it Trial?

Reply:
it's just a little odd since it's like the "TrialRunner" has "Trials" which each has their own "Runners"... we can deal with this later.

Reply:
Fixed